web/docs/enums/FlowDirection.md

```diff
@@ -15,3 +15,11 @@ ___
 • **Egress** = ``"1"``
 
 Outgoing traffic, from node observation point
+
+___
+
+### Inner
+
+• **Inner** = ``"2"``
+
+Inner traffic, ie. same source and destination node
```

Comment:
```suggestion
Inner traffic, which is the same as source and destination node
```

Reply:
Can we avoid i.e.? I'm not sure if this is actually "the same as" but just a suggestion. Feel free to ammend however you can to avoid using i.e.

Reply:
here this isn't an example, but a clarification, so "such as" doesn't work, need to find something else